pkgs/applications/networking/instant-messengers/element/element-web.nix

```diff
@@ -31,6 +32,11 @@ mkYarnPackage rec {
     sha256 = pinData.webSrcHash;
   };
 
+  jitsi_external_api = fetchurl {
+    url = "https://meet.element.io/libs/external_api.min.js";
```

Comment:
url without versions are likely asking for troubles.

Reply:
Can we just patch the script tag to reference this url?

Reply:
Not sure. I'll test. Or just vendor the script. It's pretty small.

Reply:
I thought it's better to verdor it. I don't want to tell element.io every time I use jitsi. I guess it's compiled from https://github.com/jitsi/jitsi-meet/blob/master/modules/API/external/external_api.js.